spark/graphar/src/main/java/com/alibaba/graphar/info/EdgeInfo.java

```diff
@@ -33,52 +38,37 @@ public class EdgeInfo {
     private final long chunkSize;
     private final long srcChunkSize;
     private final long dstChunkSize;
+    private final boolean directed;
     private final String prefix;
-    private final List<AdjacentList> adjacentLists;
-    private final List<PropertyGroup> propertyGroups;
+    private final Map<AdjListType, AdjacentList> adjacentLists;
+    private final PropertyGroups propertyGroups;
     private final String version;
 
     public EdgeInfo(
-            EdgeTriple edgeTriple,
+            String srcLabel,
+            String edgeLabel,
+            String dstLabel,
             long chunkSize,
             long srcChunkSize,
             long dstChunkSize,
+            boolean directed,
```

Comment:
Hi, @Thespica . It seems that there could be a method to get this `directed` value.

Reply:
Thanks, added.